### ISSUE TITLE
'density' isnt a valid attribute for mdf 'material'

### ISSUE THREAD

**Issue by Tardis50** [niess/pumas#2]
There appears to be a mismatch between the mdf format used in pumas-materials and pumas. The StandardRock (and many others) material contains a density attribute which doesn't seem to be handled by pumas properly. This raises an error on line 7536 "ERROR_REGISTER_INAVLID_XML_ATTRIBUTE" as there is no `else if (strcmp(attr, "density") == 0)` like there is on line 7560.
```
else if (node->key == MDF_KEY_MATERIAL) {
        if (strcmp(attr, "name") == 0)
                node->at1.name = value;
        else if (strcmp(attr, "file") == 0)
                node->at2.file = value;
        else
                return ERROR_REGISTER_INVALID_XML_ATTRIBUTE(
                    attr, "<material>", mdf->mdf_path,
                    mdf->line);
```
The StandardRock.xml for reference
![image](https://user-images.githubusercontent.com/20161407/106992746-4cd64700-67cd-11eb-90a7-3d8d4eb3f24b.png)

I could successfully run load.c when i added the following lines to the "material" else if statement (added to line 7535):
```
else if (strcmp(attr, "density") == 0)
        node->at3.density = value;
```

I've only started testing out pumas so take my issues with a grain of salt.

**Comment by niess** [comment]
Hello,

you are perfectly right :) The current materials file correspond to the dev version of PUMAS where the density attribute was introduced to be more consistent. Note that one can still use a different bulk density at runtime if needed, e.g. to mimic rock porosity.

I'll publish a new release of PUMAS today that is consistent with the new materials description.

**Comment by niess** [comment]
@Tardis50 I finally pushed the update to the master branch and released the new version of PUMAS: v1.0 :) I also updated the wiki accordingly. The source and the materials files should be in sync now. Sorry for the disagreement and time lost.

Note that depending on your needs there is also the [pumas-luajit](https://github.com/niess/pumas-luajit) project that might be easier to use? Yet, I am still completing the documentation right now. Therefore, I didn't advertised it too much so far.
